jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/RegexPathSpec.java

```diff
@@ -18,25 +18,29 @@
 
 package org.eclipse.jetty.http.pathmap;
 
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class RegexPathSpec extends AbstractPathSpec
```

Comment:
This class should have javadoc that describes what a RegexPathSpec is: eg prefixed by "regex|" etc etc